src/main/resources/messages.properties

```
@@ -115,6 +115,12 @@ email.response-email-for-family.provider-agrees.p3=<p><strong>Provider response:
 email.response-email-for-family.provider-agrees.p4=<p><strong>Confirmation code:</strong> {0}</p>
 email.response-email-for-family.provider-agrees.p5=<p><strong>Next steps:</strong> You will receive a letter or an email about the status of your case in 13-15 business days. If you have questions, call {0}: <a href="tel:{1}">{1}</a>.</p>
 
+# Provider Confirmation email
+email.provider-confirmation.p1=<p>Hello,</p>
```

```suggestion
email.provider-confirmation.p1=<p>Hello,</p><br />
```

Because the content includes blank line between the hello and the next paragraph, I think you'll need this BR to get the extra blank line.